README.md

```diff
@@ -26,10 +30,10 @@ This diagram provides an overview of the relationships between the various entit
 
 A typical SatCon workflow might proceed roughly as follows:
 
-1. **Register a cluster to SatCon.**  This makes the cluster known to SatCon, and the API call will return the URL for a kube yaml file which can be applied to the actual cluster.  This yaml will deploy and configure the SatCon (razee) agent, which in turn will connect back to SatCon to begin managing the cluster.
-1. **Add the cluster to one or more groups.**  If no group exists within the org, you will first need to create a group.  You can then add your cluster to the group.
+1. **Register a cluster to SatCon.**  This makes the cluster known to SatCon, and the API call will return the URL for a Kubernetes yaml file which can be applied to the actual cluster.  This yaml will deploy and configure the SatCon (razee) agent, which in turn will connect back to SatCon to begin managing the cluster.
+1. **Add the cluster to one or more groups.**  If no group exists in the IBM Cloud account, you will first need to create a group.  You can then add your cluster to the group.
```

Comment:
I would prefer, `within the organization (i.e. the IBM Cloud account)` - I think that we should make sure to use both terms everywhere we use one or the other for now.